pkg/sdn/plugin/controller.go

```diff
@@ -339,7 +340,12 @@ func (plugin *OsdnNode) SetupSDN(localSubnetCIDR, clusterNetworkCIDR, servicesNe
 func (plugin *OsdnNode) SetupEgressNetworkPolicy() error {
 	policies, err := plugin.registry.GetEgressNetworkPolicies()
 	if err != nil {
-		return fmt.Errorf("Could not get EgressNetworkPolicies: %s", err)
+		if kapierrs.IsForbidden(err) {
```

Comment:
> It looks like nodes don't have permission to set labels on themselves. It could mark itself not Ready (or we could add a new NodeConditionType). But is that better than erroring out (particularly given that you'd still need to systemctl restart it after fixing the roles)?

I'd bet that nodes/status keeps labels.  I think that not erroring out could be ok, but its important for us to be able to answer, "which nodes aren't enforcing these policies".  I don't see how I can do that today without inspecting the logs for each node.